src/nvim/normal.c

```diff
@@ -7464,8 +7464,9 @@ static void nv_esc(cmdarg_T *cap)
     if (restart_edit == 0
         && cmdwin_type == 0
         && !VIsual_active
-        && no_reason)
-      MSG(_("Type  :quit<Enter>  to exit Nvim"));
+        && no_reason) {
+      MSG(_("Type  :qa!  and press <Enter> to abandon all changes and exit Nvim"));  // NOLINT(whitespace/line_length)
```

Comment:
instead of NOLINT the string can be broken into parts like this:

```c
      MSG(_("Type  :qa!  and press <Enter> to abandon all changes and exit "
            "Nvim"));
```

(this is a C feature)